rq/plan.sh

```diff
@@ -27,5 +26,31 @@ do_build() {
 
 do_install() {
   mkdir -pv "$pkg_prefix/bin"
-  install -v "$HAB_CACHE_SRC_PATH/$pkg_filename" "$pkg_prefix/bin/$pkg_name"
+  install -v "rq" "$pkg_prefix/bin/$pkg_name"
+
+  local glibc_libs=(
+    libdl.so.2
+    librt.so.1
+    libpthread.so.0
+    libc.so.6
+    ld-linux-x86-64.so.2
+    libm.so.6
+  )
+  for lib in "${glibc_libs[@]}"; do
+    build_line "Patching ELF for ${lib}"
+    patchelf --interpreter "$(pkg_path_for core/glibc)/lib/${lib}" --set-rpath "${LD_RUN_PATH}" "${pkg_prefix}/bin/rq"
+  done
+
+  local gcclibs_libs=(
+    libstdc++.so.6
+    libgcc_s.so.1
+  )
+  for lib in "${gcclibs_libs[@]}"; do
+    build_line "Patching ELF for ${lib}"
+    patchelf --interpreter "$(pkg_path_for core/gcc-libs)/lib/${lib}" --set-rpath "${LD_RUN_PATH}" "${pkg_prefix}/bin/rq"
+  done
+}
+
+do_strip() {
+  return 0
```

Comment:
Perhaps a comment here to remind us why this turned off would be good.  My guess is that we turned this off because of the following patchelf problem: https://github.com/NixOS/patchelf/issues/10

Reply:
Yep.

Or! Things seem to be happy if I move the elf patching to `do_strip()`.